community.dbuild

```diff
@@ -984,4 +989,60 @@ build += {
     extra.exclude: ["plugin"]  // we never build sbt plugins
   }
 
+  // forked to remove the usual bintray gunk
```

Comment:
with this particular plugin, I don't think so, sadly. a bunch of other projects are affected, too. the ticket is https://github.com/typesafehub/dbuild/issues/158